wsd/ClientRequestDispatcher.cpp

```diff
@@ -49,9 +54,15 @@
 #include <Poco/MemoryStream.h>
 #include <Poco/Net/HTMLForm.h>
 #include <Poco/Net/NetException.h>
+#include <Poco/Net/DNS.h>
 #include <Poco/Net/PartHandler.h>
+#if !MOBILEAPP
+#include <Poco/Net/SSLException.h>
```

Comment:
Again we don't want lots of Poco-ness - and particularly synchronous DNS is a nightmare.